administrative-sdk.js

```diff
@@ -1865,7 +1865,7 @@ class Sdk {
     var words = [];
     inWords.forEach(function(word) {
       var chunks = [];
-      word.chunks.forEach(function(chunk) {
+      word.forEach(function(chunk) {
```

Comment:
I guess this makes sense. Please revert the other change, so this can get merged.